pkg/graveler/graveler.go

```diff
@@ -1593,6 +1593,10 @@ func (g *Graveler) Merge(ctx context.Context, repositoryID RepositoryID, destina
 		if err != nil {
 			return nil, err
 		}
+		g.log.WithFields(logging.Fields{
+			"from_meta_range": fromCommit.MetaRangeID,
+			"to_meta_range":   toCommit.MetaRangeID,
+			"base_meta_range": baseCommit.MetaRangeID}).Trace("Merge")
```

Comment:
Think we need to add the repository ID, source and destination too
```suggestion
			"base_meta_range": baseCommit.MetaRangeID,
		}).Trace("Merge")
```

Reply:
Adding Thanks!